aiven/client/client.py

```diff
@@ -2134,3 +2135,410 @@ def delete_oauth2_client_secret(self, account_id: str, client_id: str, secret_id
             self.delete,
             self.build_path("account", account_id, "oauth_client", client_id, "secret", secret_id),
         )
+
+    def flink_list_applications(
+        self,
+        *,
+        project: str,
+        service: str,
+    ) -> Mapping:
+        return self.verify(
+            self.get,
+            self.build_path(
+                "project",
+                project,
+                "service",
+                service,
+                "flink",
+                "application",
+            ),
+        )
+
+    def flink_create_application(
+        self,
+        *,
+        project: str,
+        service: str,
+        application_properties: Mapping[str, str],
+    ) -> Mapping:
+        return self.verify(
+            self.post,
+            self.build_path(
+                "project",
+                project,
+                "service",
+                service,
+                "flink",
+                "application",
+            ),
+            body=application_properties,
+        )
+
+    def flink_get_application(
+        self,
+        *,
+        project: str,
+        service: str,
+        application_id: str,
+    ) -> Mapping:
+        try:
+            application_uuid = UUID(application_id)
+        except ValueError as e:
+            raise ValueError("Application id must be a valid UUID") from e
```

Comment:
Better would be to have the exact param name in the error, e.g.`application_id not a valid UUID`

Is it needed to parse and format as string the `application_id`? The REST API will respond with an error if `application_id` is not an uuid? There's no validation on the client side of other parameters, e.g. `project` has certain naming convention, but it's not checked here - why check `application_id`?

Reply:
For consistency, removed the validation from client.